backend/import/.test_data/mo/http:##morademo.atlas.magenta.dk#service#e#0014209a-8832-44b3-b7a0-4cc209a71993#details#.json

```diff
@@ -0,0 +1 @@
+{"address": true, "association": false, "engagement": true, "it": false, "leave": false, "manager": false, "org_unit": false, "role": false}
```

Comment:
You should probably for these files, to make them ever so slightly less unreadable. `python -m json.tool` is nice.

Reply:
The intent is, though, not that they should be read by humans but that they should store actual results of `request.get().json()`. `json.dump(...)` may indeed be too barebones, though.

Reply:
```python
json.dumps(obj, indent=2)
```

This is, essentially, what `python -m json.tool` does…